drivers/SmartThings/zwave-switch/src/test/test_zooz_zen_30_dimmer_relay_preferences.lua

```diff
@@ -0,0 +1,531 @@
+local test = require "integration_test"
+local t_utils = require "integration_test.utils"
+local zw = require "st.zwave"
+local zw_test_utils = require "integration_test.zwave_test_utils"
+local Configuration = (require "st.zwave.CommandClass.Configuration")({ version = 4 })
+
+local zooz_zen_dimmer_relay_endpoints = {
+  {
+    command_classes = {
+      { value = zw.METER },
+      { value = zw.BASIC },
+      { value = zw.SWITCH_BINARY },
+      { value = zw.SWITCH_MULTILEVEL }
+    }
+  }
+}
+
+local zooz_zen_dimmer_relay = test.mock_device.build_test_zwave_device({
+  profile = t_utils.get_profile_definition("zooz-zen-30-dimmer-relay.yml"),
+  zwave_endpoints = zooz_zen_dimmer_relay_endpoints,
+  zwave_manufacturer_id = 0x027A,
+  zwave_product_type = 0xA000,
+  zwave_product_id = 0xA008
+})
+
+local function test_init()
+  test.mock_device.add_test_device(zooz_zen_dimmer_relay)
+end
+test.set_test_init_function(test_init)
+
+do
```

Comment:
Instead of manually writing each one of these tests, could you just define them in a loop by pulling from the preferences file? I believe `1` is a legal value for all of the preferences.